docs/root/configuration/listeners/network_filters/zookeeper_proxy_filter.rst

```diff
@@ -76,64 +79,119 @@ following counters are available:
 
 Per opcode latency statistics
 -----------------------------
 
-The filter will gather latency statistics in the *<stat_prefix>.zookeeper.<opcode>_response_latency* namespace.
+The filter will gather latency statistics in the *<stat_prefix>.zookeeper.<opcode>_resp_latency* namespace.
 Latency stats are in milliseconds:
 
 .. csv-table::
   :header: Name, Type, Description
   :widths: 1, 1, 2
 
-  connect_response_latency, Histogram, Opcode execution time in milliseconds
-  ping_response_latency, Histogram, Opcode execution time in milliseconds
-  auth_response_latency, Histogram, Opcode execution time in milliseconds
+  connect_resp_latency, Histogram, Opcode execution time in milliseconds
+  ping_resp_latency, Histogram, Opcode execution time in milliseconds
+  auth_resp_latency, Histogram, Opcode execution time in milliseconds
   watch_event, Histogram, Opcode execution time in milliseconds
-  getdata_response_latency, Histogram, Opcode execution time in milliseconds
-  create_response_latency, Histogram, Opcode execution time in milliseconds
-  create2_response_latency, Histogram, Opcode execution time in milliseconds
-  createcontainer_response_latency, Histogram, Opcode execution time in milliseconds
-  createttl_response_latency, Histogram, Opcode execution time in milliseconds
-  setdata_response_latency, Histogram, Opcode execution time in milliseconds
-  getchildren_response_latency, Histogram, Opcode execution time in milliseconds
-  getchildren2_response_latency, Histogram, Opcode execution time in milliseconds
-  getephemerals_response_latency, Histogram, Opcode execution time in milliseconds
-  getallchildrennumber_response_latency, Histogram, Opcode execution time in milliseconds
-  remove_response_latency, Histogram, Opcode execution time in milliseconds
-  exists_response_latency, Histogram, Opcode execution time in milliseconds
-  getacl_response_latency, Histogram, Opcode execution time in milliseconds
-  setacl_response_latency, Histogram, Opcode execution time in milliseconds
-  sync_response_latency, Histogram, Opcode execution time in milliseconds
-  multi_response_latency, Histogram, Opcode execution time in milliseconds
-  reconfig_response_latency, Histogram, Opcode execution time in milliseconds
-  close_response_latency, Histogram, Opcode execution time in milliseconds
-  setauth_response_latency, Histogram, Opcode execution time in milliseconds
-  setwatches_response_latency, Histogram, Opcode execution time in milliseconds
-  setwatches2_response_latency, Histogram, Opcode execution time in milliseconds
-  checkwatches_response_latency, Histogram, Opcode execution time in milliseconds
-  removewatches_response_latency, Histogram, Opcode execution time in milliseconds
-  check_response_latency, Histogram, Opcode execution time in milliseconds
+  getdata_resp_latency, Histogram, Opcode execution time in milliseconds
+  create_resp_latency, Histogram, Opcode execution time in milliseconds
```

Comment:
Changing the names of the stat names may cause a breaking change if some 3rd party is dependent on them (throughout this document and PR).
@htuch I'm not sure if stat renaming is typically considered an API breaking change or not, but are you aware of prior cases where stat names were renamed/deprecated?

Reply:
There is prior art. I think we don't apply API breaking change policy, but we do put this in the class of "don't break operators gratuitously", so for a simple rename for consistency we might avoid that, and anything more major might roll out under runtime protection or something like that. @mattklein123 probably remembers better than I do the case history and philosophy.

Reply:
Thank you for the clarification! I've reverted all changes related to the existing metric names. For all changes in the .rst file, I explained in my previous comment above.